projects/intervyoume.md

```diff
@@ -0,0 +1,14 @@
+IntervYouMe
+-----------
+
+### Background
+
+IntervYouMe will be a platform that allows employers to connect with potential employees, face to face, from anywhere across the world. Our service will simplify and expedite both the interview and hiring process, which in turn may significantly impact the incessantly high rate of unemployment on a global scale. IntervYouMe, employment made easy.
+
+### Challenge
+
+With the sad and ever-present concept of unemployment, there will always be a demand in the interview and employment market. How can we go about creating a simple yet efficient way for users to conduct personal and technical interviews on a global scale?
+
+### Solution
+
+Private - To be discussed in class.
```

Comment:
I think if you want the project to be accepted you need to write something here.